pkgs/by-name/ew/eww/package.nix

```diff
@@ -17,13 +19,25 @@ rustPlatform.buildRustPackage rec {
   src = fetchFromGitHub {
     owner = "elkowar";
     repo = "eww";
-    rev = "refs/tags/v${version}";
+    rev = "v${version}";
```

Comment:
remove this change

Reply:
Can you consider explaining why? fetchFromGitHub can find the tag just fine if I leave it like this.

Reply:
There's absolutely no reason for this change. And it's better to be explicit especially if branches get created in the future as well as for the sake of update scripts.

Reply:
Reverted, thanks.